src/implementations/vanilla/components/tag/tag.scss

```diff
@@ -0,0 +1,96 @@
+/**
+ * Tags
+ * @define tag
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/icon';
+@use '@ecl/theme-dev/maps/spacing';
+@use '@ecl/theme-dev/maps/typography';
+
+.ecl-tag {
+  align-items: center;
+  background-color: map.get(color.$color, 'grey-15');
+  border-radius: (
+      2 * map.get(spacing.$spacing, 'xs') +
```

Comment:
why this indentation here? was it formatted like this by the linter?

Reply:
indeed, I didn't change it, so that's the linter preference